### PR TITLE
Feature/empty state ng content

### LINKED ISSUE #187: Empty state title and description should support ng-content.
#### Describe the desired behavior
Empty state title and description should support ng-content.

#### Describe the current behavior
Empty state title and description only support string type input.

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Add ability to use ng-content for title and description in empty-state component.

Fixes #187

### CODE REVIEW

**Comment by huayunh** [comment]
Changelog? Code seems good

File: components/src/core/empty-state/empty-state.component.spec.ts [hunk 1]
```diff
@@ -58,36 +58,14 @@ describe('Empty State Component', () => {
...
-        void expect(warnSpy).toHaveBeenCalledTimes(1);
-    });
-
-    it('should not show empty title', () => {
-        component.title = undefined;
-        spyOn(console, 'warn').and.stub();
-        fixture.detectChanges();
-        const titleElement = fixture.debugElement.query(By.css('h2'));
-        void expect(titleElement.nativeElement.innerHTML).toBeFalsy();
+        void expect(titleElement.nativeElement.innerHTML).toContain('title');
```
Comment (joebochill) [inline components/src/core/empty-state/empty-state.component.spec.ts:61]: Can we update these tests to test the new scenarios/usage?

File: docs/EmptyState.md [hunk 2]
```diff
@@ -53,10 +53,12 @@ The following child elements are projected into `<pxb-empty-state>`:
```
Comment (joebochill) [inline docs/EmptyState.md:53]: ^^ Title is no longer a required @Input ^^

**Review by joebochill — APPROVED** [review]

File: docs/EmptyState.md (hunk 2 above)
Comment (emclaug2) [reply docs/EmptyState.md:53]: Good catch, I've updated the docs and will merge this PR in.

**Pull request merged**